README.md

```diff
@@ -42,8 +42,7 @@ match '/saml/logout' => 'saml_idp#logout', via: [:get, :post, :delete]
 Create a controller that looks like this, customize to your own situation:
 
 ``` ruby
-class SamlIdpController
-  include SamlIdp::IdpController
+class SamlIdpController < SamlIdp::IdpController
```

Comment:
@pkarman Happy to help. So I believe inherit the class `SamlIdp::IdpController` is the correct documentation right. Would we merging that in master anytime soon. It create lot of confusion to the new comers.